test/test_security_service.py

```diff
@@ -103,7 +99,7 @@ def test_crud_security_group_service(self):
         sg = self.provider.security.security_groups.create(
             name=name, description=name)
         #Empty security groups don't exist in GCE. Let's add a dummy rule.
```

Comment:
Re. the change of the signature for SG's `create` method - adding the `initialRules` parameter sounds like a good idea; I don't see a realistic usecase for an empty SG anyhow.